### PR TITLE
CODENVY-568: Fire FactoryAccepted Event then factory accepted: poject imported, actions perform   

### PR DESCRIPTION
**Pull request by vparfonov**

### CODE REVIEW

**Comment by vparfonov** [comment]
@skabashnyuk @sleshchenko @mshaposhnik

File: plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/accept/AcceptFactoryHandler.java [hunk 1]
```diff
@@ -25,10 +26,12 @@
 import org.eclipse.che.api.factory.shared.dto.Ide;
 import org.eclipse.che.ide.api.action.ActionManager;
 import org.eclipse.che.ide.api.app.AppContext;
+import org.eclipse.che.ide.api.factory.FactoryAcceptedEvent;
 import org.eclipse.che.ide.api.machine.events.WsAgentStateEvent;
 import org.eclipse.che.ide.api.machine.events.WsAgentStateHandler;
 import org.eclipse.che.ide.api.notification.NotificationManager;
 import org.eclipse.che.ide.api.notification.StatusNotification;
+import org.eclipse.che.ide.util.loging.Log;
```
Comment (sleshchenko) [inline plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/accept/AcceptFactoryHandler.java]: Log import is unused

File: plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/accept/AcceptFactoryHandler.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 import com.codenvy.ide.factory.client.FactoryLocalizationConstant;
 import com.codenvy.ide.factory.client.utils.FactoryProjectImporter;
+import com.google.gwt.user.client.Timer;
```
Comment (sleshchenko) [inline plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/accept/AcceptFactoryHandler.java]: unused import

**Comment by sleshchenko** [comment]
other LGTM

**Comment by mshaposhnik** [comment]
OK

**Comment by skabashnyuk** [comment]
ок

**Pull request merged**